src/main/features/dashboard/views/macro/partyDetails.njk

```diff
@@ -0,0 +1,29 @@
+{% macro otherPartyDetails(otherParty, partyClaimData) %}
+
+  <h2 class="heading-medium no-top-margin">{{ otherParty }}'s details</h2>
+  <table class="govuk-table">
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell--left">Name</td>
+        <td class="govuk-table__cell--right">{{ partyClaimData.name }}</td>
+      </tr>
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell--left">Address</td>
+        <td class="govuk-table__cell--right">{{ partyClaimData.address | formatAddress | default('Not available') | safe }}</td>
```

Comment:
any specific reason for using 'safe' here?. this is needed only if absolutely needed

Reply:
if safe is not used, data appears as simple text showing HTML code, safe renders code as DOM elements rather than plain text